utils/storybook/CustomTheme.ts

```diff
@@ -0,0 +1,22 @@
+export const customThemePallete = {
+  palette: {
+    primary: {
+      main: 'purple',
+    },
+    alert: {
+      main: 'coral',
+    },
+    error: {
+      main: 'crimson',
+    },
+    success: {
+      main: 'aquamarine',
+    },
+    neutral: {
+      main: 'gray',
+    },
+    common: {
+      focusOutline: 'purple',
+    },
+  },
```

Comment:
This file can grow and be edited, rather than having multiple custom theme's per story.